src/frm_main.cpp

```diff
@@ -99,8 +99,10 @@ bool FrmMain::initSDL(const CmdLineSetup_t &setup)
     //Write into log the application start event
     pLogDebug("<Application started>");
 
+#ifndef NO_SCREENSHOT
     m_screenshotPath = AppPathManager::screenshotsDir();
     m_gifRecordPath = AppPathManager::gifRecordsDir();
+#endif
```

Comment:
Yeah, on Emscripten they are fully useless as no way to download them developed right now